src/main.rs

```diff
@@ -2,9 +2,15 @@ use geo::Point;
 use serde_json::to_string_pretty;
 use zonebuilder::{clockboard, Params};
 
+
 fn main() {
+
+    // eprintln!("{:#?}", Params::default());
+
     let gj = clockboard(Point::new(0.0, 0.0), Params::default());
```

Comment:
The problem is you're passing in `Params::default()`, so there's no CLI argument parsing happening at all. You want `Params::from_args()`.

Reply:
:+1: now updated so it's passing `Params::from_args()` as suggested. However, it's now giving this error message which I don't understand, any ideas?

```
cargo run                                                       
    Finished dev [unoptimized + debuginfo] target(s) in 0.04s
     Running `target/debug/zonebuilder`
error: Invalid value for '--distances <distances>...': invalid float literal
```